packages/framework/src/Model/Product/ProductFacade.php

```diff
@@ -354,9 +354,26 @@ public function getAllProductSellingPricesIndexedByDomainId(Product $product)
         $productSellingPrices = [];
 
         foreach ($this->pricingGroupRepository->getAll() as $pricingGroup) {
```

Comment:
Please, iterate over `$this->domain->getAllIds()`. Otherwise you'd call the `getAllProductSellingPricesByDomainId()` method multiple times for every domain that has more than one `PricingGroup` assigned.

Reply:
You are right, done, thx